pkg/mountmanager/mount_linux.go

```diff
@@ -63,7 +63,7 @@ func (m *NodeMounter) GetSafeFormatAndMount() *mount.SafeFormatAndMount {
 
 // Resize returns boolean and error if any
 func (m *NodeMounter) Resize(devicePath string, deviceMountPath string) (bool, error) {
-	r := mount.NewResizeFs(m.NewSafeFormatAndMount().Exec)
+	r := mount.NewResizeFs(m.GetSafeFormatAndMount().Exec)
```

Comment:
This change seems correct as it ensures we get the instance which holds command fakes instead of recreating it an loosing the fakes.